packages/peregrine/lib/talons/FilterModal/CurrentFilters/useCurrentFilters.js

```diff
@@ -0,0 +1,24 @@
+import { useCallback } from 'react';
+import { useCatalogContext } from '@magento/peregrine/lib/context/catalog';
+
+export const useCurrentFilters = props => {
+    // TODO: Replace with useRouter.
+    const { history, location } = props;
+
+    const [{ chosenFilterOptions }, { removeFilter }] = useCatalogContext();
```

Comment:
I noticed we don't hook up the `addFilter` from the catalog actions. Can the filter related actions be removed from catalog? I think we can use your new `removeItem` filter action.

Reply:
Yes, that's correct. Filter state is no longer connected to Redux at all. I need to remove that stuff.

Reply:
All references to Redux state have been removed. I also removed all of the effects that formerly had to check for filter params.